af/fastpath/fingerprints.csv

```diff
@@ -0,0 +1,58 @@
+12,KR,http://warning.or.kr,,,
+13,RO,Accesul dumneavoastră către acest site a fost restricționat,,,
+14,GF,"xtpage = ""page-blocage-terrorisme""",,,
+15,ID,,Location,http://internet-positif.org,
+17,RU,,Location,http://warning.rt.ru,
+18,SA,,Server,Protected by WireFilter,
+19,AE,,Server,Protected by WireFilter,
+20,SD,,Location,,http://196.1.211.6:8080/alert/
+21,QA,,Location,,http://www.vodafone.qa/alu.cfm
+22,KR,,Location,,http://www.warning.or.kr
+23,PT,,Location,,http://mobilegen.vodafone.pt/denied/dn
+24,NO,,Location,,http://block-no.altibox.net/
+16,GB,,Location,http://blocked.nb.sky.com,
+25,ZZ,,Server,,Kerio Control Embedded Web Server
+26,SG,,Location,,http://www.starhub.com:80/personal/broadband/value-added-services/safesurf/mda-blocked.html
+27,PH,,Location,http://surfalert.globe.com.ph/usedpromo?dest_url,
+28,PH,,Location,,http://cube.sunbroadband.ph:8020/balanceToOffer/init
+29,EG,,Location,http://notification.etisalat.com.eg/etisalat/notification/redirectionactions.html?,
+30,KE,,Location,http://159.89.232.4/alert,
+31,GB,,Location,,http://ee.co.uk/help/my-account/corporate-content-lock
+32,GB,,Location,http://Filter7external.schoolsbroadband.co.uk/access,
+33,ZZ,,Location,https://account.nowtv.com/broadband-buddy/blocked-pages/?domain=,
+34,GB,,Location,,www.vodafone.co.uk/contentcontrolpage/vfb-category-blocklist.html
+35,GB,,Location,,http://three.co.uk/mobilebroadband_restricted
+36,CH,,Location,https://192.168.88.1/sgerror.php,
+37,ZZ,,Location,http://1.2.3.50/ups/no_access,
+38,VN,,Location,http://ezxcess.antlabs.com/login/index.ant?url,
+39,ID,"If you find, site that you want to access does not include any of the above categories, please email to",,,
+40,FR,,Via,,1.1 proxy2.rmc.local:80 (Cisco-WSA/9.1.1-074)
+41,DK,,Via,,1.1 dsbpx001.dsb.dk:25 (Cisco-WSA/9.1.1-074)
+42,DE,,Via,,1.1 s690-10.noc.rwth-aachen.de:80 (Cisco-WSA/10.1.1-230)
+43,DE,,X-App-Url,,https://blocked.netalerts.io
+44,BR,,Via,,1.1 wsa07.grupoamil.com.br:80 (Cisco-WSA/9.1.2-010)
+45,AU,,Via,,1.1 ac2106-wsag2.wsmartwifi.com.au:80 (Cisco-WSA/10.1.1-235)
+46,ZZ,,Location,http://www.webscanningservice.com/WebServicesAlertPage/WebURLAlert.aspx,
+47,AU,,Location,https://go.telstra.com.au/broadbandprotect/networkprotectionstandard,
+48,AR,,Via,,1.1 FC-WSA-FibertelZone3.int.fibercorp.com.ar:80 (Cisco-WSA/10.1.0-204)
+49,US,,Via,,1.1 MLD-C-Barracuda.mld.org (http_scan_byf/3.5.16)
+50,US,,Via,,1.1 forcepoint-wcg.chambersburg.localnet
+51,US,,Location,http://filter.esu9.org:8080/webadmin/deny/index.php,
+52,US,,Location,http://ibossreporter.edutech.org/block/bp.html,
+53,US,,Location,http://alert.scansafe.net/alert/process,
+54,US,,Location,http://184.168.221.96:6080/php/urlblock.php,
+55,US,,Location,https://gateway.wifast.com:443/wifidog/login/,
+56,US,,Location,https://mpswebfilterwashbu.mpls.k12.mn.us:6082/php/uid.php,
+57,PA,,Via,,1.1 barracuda.tropigas.com.pa (http_scan_byf/3.3.1)
+58,BE,,Via,,1.1 webfilter.stjohns.net (http_scan_byf/3.5.16)
```

Comment:
Is this a duplicate of the fingeprints inside of `utils.py` in the dict?

Reply:
Yes, dropped.